perl/README.md

```diff
@@ -0,0 +1,1021 @@
+# Bystro High Dimensional Genomic Annotator Documentation
+
+## What is the Bystro Annotator?
+
+The Bystro Annotator is the fastest and most comprehensive data curation and labeling library in the world for genetic data. It takes 1 or more VCF ([Variant Call Format](https://samtools.github.io/hts-specs/VCFv4.2.pdf)) or SNP ([PEMapper/PECaller](https://www.pnas.org/doi/full/10.1073/pnas.1618065114)) files as input, and outputs a cleaned and thoroughly labeled (annotated) representation of your data, along with a genotype dosage matrix in the [Arrow Feather V2/IPC format](https://arrow.apache.org/docs/python/feather.html), and a set of statistics on the data.
+
+Bystro Annotator processes both variant and sample genotypes. It is capable of processing millions of samples and billions of mutations on commodity hardware such as a laptop or a workstation. It is roughly **100,000** times faster than [Variant Effect Predictor](https://www.ensembl.org/info/docs/tools/vep/index.html) (VEP), **100** times faster than [Annovar](https://doc-openbio.readthedocs.io/projects/annovar/en/latest/), and **50** times faster than [Illumina Connected Annotations](https://developer.illumina.com/illumina-connected-annotations), all while outputting more annotations than any of these tools. What takes VEP years to do, Bystro can do in minutes to hours, all without requiring multiple servers.
+
+Bystro's performance isn't just about speed, it's also about comprehensiveness. For instance, because of its performance, Bystro can afford to provide complete annotations from gnomad v4.1, for all populations and subpopulations, from the exomes, genomes, and joint datasets, genome-wide. This is intersecting terabytes of data over the entire genome with each variant, all done in microseconds per variant. **No other tool can do this**.
+
+## Running Your First Annotation
+
+See the [INSTALL.md#configuring-the-bystro-annotator](./INSTALL.md#configuring-the-bystro-annotator) section for instructions on how to configure the Bystro Annotator
+
+```sh
+bystro-annotate.pl --config ~/bystro/config/hg38.yml --threads 32 --input gnomad.genomes.v4.0.sites.chr22.vcf.bgz --output test/my_annotation --compress gz
+```
+
+The above command will annotate the `gnomad.genomes.v4.0.sites.chr22.vcf.bgz` file with the hg38 database, using 32 threads, and output the results to `test`, and will use `my_annotation` as the prefix for output files.
```

Comment:
Do we have a link to this gnomad chr22 set so they can follow along with the tutorial?